spec/services/pii/nist_encryption_spec.rb

```diff
@@ -49,70 +49,69 @@
   describe 'KMS encryption of random R' do
     it 'creates encrypted key D and hash E' do
       random_R, ciphered_R = stub_aws_kms_client
-      allow(Pii::Cipher).to receive(:random_key).and_return(random_R)
+      allow(SecureRandom).to receive(:random_bytes).and_return(random_R)
```

Comment:
looks like you're moving away from `Pii::Cipher.random_key` in favor of just calling `SecureRandom.random_bytes` instead. Looks like the only other place where this is used is in `nist_encryption_spec:34`, maybe remove it completely?

Reply:
Yeah, it just forwards to `random_key` and doesn't actually do anything related to being a cipher so I decided to move that out. Yes, I'll go back and remove it entirely.